# generated by patch-package 6.5.0 on 2021-09-27 05:10:58
#
# command:
#   npx patch-package web3-core-method
#
# declared package:
#   web3-core-method: 1.5.2
#
diff --git a/node_modules/web3-core-method/lib/index.js b/node_modules/web3-core-method/lib/index.js
index 7dd2cfe..e31fb91 100644
--- a/node_modules/web3-core-method/lib/index.js
+++ b/node_modules/web3-core-method/lib/index.js
@@ -437,7 +437,7 @@ Method.prototype._confirmTransaction = function (defer, result, payload) {
     // start watching for confirmation depending on the support features of the provider
     var startWatching = function (existingReceipt) {
         const startInterval = () => {
-            intervalId = setInterval(checkConfirmation.bind(null, existingReceipt, true), 1000);
+            intervalId = setInterval(checkConfirmation.bind(null, existingReceipt, true), 30000);
         };
         if (!this.requestManager.provider.on) {
             startInterval();
diff --git a/node_modules/web3-core-method/src/index.js b/node_modules/web3-core-method/src/index.js
index fddedbd..ef406b8 100644
--- a/node_modules/web3-core-method/src/index.js
+++ b/node_modules/web3-core-method/src/index.js
@@ -548,7 +548,7 @@ Method.prototype._confirmTransaction = function (defer, result, payload) {
     // start watching for confirmation depending on the support features of the provider
     var startWatching = function (existingReceipt) {
         const startInterval = () => {
-            intervalId = setInterval(checkConfirmation.bind(null, existingReceipt, true), 1000);
+            intervalId = setInterval(checkConfirmation.bind(null, existingReceipt, true), 30000);
         };
 
         if (!this.requestManager.provider.on) {
